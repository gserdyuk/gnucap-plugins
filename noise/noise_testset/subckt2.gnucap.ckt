A 5-R noise circuit
*
*  +- R1-+---
*  |         |
*  V         RL
*  |         | 
* ---       ---
*
.subckt res_sub in out
R1  in out 100
.ends

Vin 1 0 ac 1 dc 0
X1  1 3 res_sub
RL  3 0 5.K

.noise v(3) vin dec 1 10. 100 
.end
