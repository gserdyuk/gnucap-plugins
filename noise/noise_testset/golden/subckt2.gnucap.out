Gnucap gnucap-arails  2009.12.07 RCS 26.136
The Gnu Circuit Analysis Package
Never trust any version less than 1.0
Copyright 1982-2009, Albert Davis
Gnucap comes with ABSOLUTELY NO WARRANTY
This is free software, and you are welcome
to redistribute it under the terms of 
the GNU General Public License, version 3 or later.
See the file "COPYING" for details.
A 5-R noise circuit 
#Freq inoise_density onoise_density
 10.  1.69144E-18  1.62576E-18
 100.  1.69144E-18  1.62576E-18

#index inoise_total onoise_total
0  152.23E-18  146.318E-18

